crowbar_framework/app/models/backup.rb

```diff
@@ -167,6 +171,7 @@ def save_archive
     self.version = meta["version"]
     self.size = path.size
     self.created_at = Time.zone.parse(meta["created_at"])
+    self.migration_level = meta["migration_level"]
```

Comment:
Please add a fallback: this patch won't be in the release, so we need to assume there'll be backups without this metadata.